src/main/java/core/basesyntax/service/DataWriterService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface DataWriterService {
+    void writeReportToFile(String report, String fileName);
```

Comment:
we should not use 'Report' in abstraction
```suggestion
    void writeToFile(String content, String fileName);
```